components/segmenter/src/line.rs

```diff
@@ -963,6 +979,25 @@ impl<'l, 's, Y: LineBreakType<'l, 's>> Iterator for LineBreakIterator<'l, 's, Y>
                 // I may have to fetch text until non-SA character?.
             }
 
+            // LB15a has to read previous data before.
```

Comment:
Why is this handled in code rather than in the state table like LB14, LB30a, or LB28a part 3, which all require multiple characters of context before?

Note that the rule is written in such a way that you do not actually need to look across breaks—precisely so that it is implementable by a state machine that advances from break to break—; sot and the various things that could have led to a break (including hard line breaks) are treated the same.

Contrary to LB8a, LB9, and LB10, rule LB15a is tailorable; hardcoding it means users cannot tailor it.